release_dashboard/lib/main.dart

```diff
@@ -15,7 +15,7 @@ import 'state/status_state.dart';
 import 'widgets/clean_release_button.dart';
 import 'widgets/progression.dart';
 
-const String _title = 'Flutter Desktop Conductor (Not ready, do not use)';
+const String _title = 'Flutter Desktop Conductor';
 
 Future<void> main() async {
```

Comment:
You can pass List<String> args here, and add `packages:args` to use a command line flag to flip the isDev instead

Reply:
Added a todo.